### ISSUE TITLE
we want to integrate this in quilljs and want to use latex keyboard for math and science

### ISSUE THREAD

**Issue by ngcgolf11** [mathquill/mathquill#844]
we want to integrate this in quilljs and want to use virtual latex keyboard for math and science please help us mark this on highest priority and suggest us something

**Comment by RobbinBaauw** [comment]
I don't think you know how open source works... Anyway, there is a library which can help with that: https://github.com/c-w/mathquill4quill. It works, perhaps needing some changes, look at our version here https://github.com/RobbinBaauw/CSHub/blob/dev/cshub-client/src/plugins/quill/mathquill4quill.min.js
